docs/pages/switch.js

```diff
@@ -57,6 +57,196 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
+        </Label>
+      </Flex.Item>
+      <Switch
+        onChange={() => setSwitched(!switched)}
+        id="makesecret"
+        switched={switched}
+      />
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Truncate label text, instead allow it to wrap to form another line. "
+            defaultCode={`
+function SwitchExample3() {
+  const [switched, setSwitched] = React.useState(false);
+
+  return (
+    <Flex gap={2} alignItems="center">
+      <Switch
+        onChange={() => setSwitched(!switched)}
+        id="setboard"
+        switched={switched}
+      />
+      <Label htmlFor="setboard">
+        <Text>Set board to...</Text>
+      </Label>
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Communicate why a switch is disabled and how to enable it if possible."
+            defaultCode={`
+<Flex gap={2} alignItems="center">
+  <Switch
+    onChange={() => {}}
+    id="showsecretboards"
+    switched={false}
+    disabled={true}
+  />
+  <Label htmlFor="showsecretboards">
+    <Flex direction="column" gap={1}>
+      <Text color="subtle">Show secret boards</Text>
+      <Text size="100" color="subtle">You don't have any secret boards</Text>
+    </Flex>
+  </Label>
+</Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use alternative styling to represent the functionality of a switch. Use Switch instead."
+            defaultCode={`
+<Flex direction="column" gap={2} width="100%" maxWidth={300}>
+  <Heading size="300" accessibilityLevel={3}>On Pinterest notifications</Heading>
+  <Flex direction="column" gap={4}>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Text>Comments</Text>
+      </Flex.Item>
+      <Icon icon="check-circle" color="default" accessibilityLabel="checked circle" />
+    </Flex>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Text>Mentions</Text>
+      </Flex.Item>
+      <Icon icon="check-circle" color="default" accessibilityLabel="checked circle" />
+    </Flex>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Text>Views</Text>
+      </Flex.Item>
+      <Icon icon="circle-outline" accessibilityLabel="unchecked circle" />
+    </Flex>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Text>Saves</Text>
+      </Flex.Item>
+      <Icon icon="check-circle" color="default" accessibilityLabel="checked circle" />
+    </Flex>
+  </Flex>
+</Flex>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use a standalone switch if the surrounding context is clear without an associated label. However, be sure to include a visually hidden label for screen readers."
```

Comment:
Apologies, I think I missed this in the dropbox

We may want to do "Use a standalone switch if the surrounding context is clear without an immediately adjacent label. However, be sure to pair the Label with the Switch using the `htmlFor` and `id` props." instead

Reply:
This totally works for me. @khalvy, whaddaya think?

Reply:
Switch doesnt actually have a label prop so, this is mainly just stating that the label doesn't have to be right next to the switch, and just to make sure to link them properly